cmd/dep/testdata/gom/Gomfile

```diff
@@ -0,0 +1,2 @@
+gom "github.com/sdboyer/deptest", :commit => "3f4c3bea144e112a69bbe5d8d01c1b09a544253f"
```

Comment:
I notice that this only has testdata for `:commit` and not `:tag` or `:branch`. Since we also need an integration test for `dep init` importing a gomfile, [see godep for an integration test example](https://github.com/golang/dep/tree/master/cmd/dep/testdata/harness_tests/init/godep/case1), you could include those in that new integration test.